stanford/playbooks/analytics-api.yml

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env ansible-playbook
+---
+- include: ../playbooks/cluster.yml
+  vars:
+    CLUSTER_NAME: analytics-api
+  when: redeploy | default(False) | bool
+
+- name: Deploy analytics-api service
+  hosts: "tag_Cluster_{{ CLUSTER_NAME }}:&tag_todo_deploy:&tag_Deployment_{{ COMMON_DEPLOYMENT }}:&tag_Number_{{ CLUSTER_NUMBER }}"
+  gather_facts: True
+  become: True
+  vars:
+    CLUSTER_NAME: analytics-api
+    migrate_db: "yes"
+    disable_edx_services: false
+    ENABLE_DATADOG: False
+    ENABLE_SPLUNKFORWARDER: False
+    ENABLE_NEWRELIC: False
+  vars_files:
+    - "{{ COMMON_DEPLOYMENT_DIR }}/common.yaml"
+    - "{{ COMMON_DEPLOYMENT_DIR }}/insights.yml"
```

Comment:
Which variables does `data-api` actually depend on from this?

Typically, the roles are fairly well segmented and only need their prefixed vars,
`INSIGHTS_*` for `insights`, `ANALYTICS_API_*` for `data-api`, etc..

Reply:
I'm going to try removing it and see what happens.